pkgs/desktops/gnome/extensions/guillotine/default.nix

```diff
@@ -0,0 +1,54 @@
+{
+  stdenv,
+  lib,
+  fetchFromGitLab,
+  glib
+}:
+
+stdenv.mkDerivation rec {
```

Comment:
Most people nowadays supply `mkDerivation` a function that has the `finalAttrs` argument. It's consider better than using `rec` in such a large scope. It also makes overriding easier.